### PR TITLE
Fine-tune Renovate auto-merge for layoutlib

### PR DESCRIPTION
**Pull request by TWiStErRob**
* Follow-up on #1168

### CODE REVIEW

File: renovate.json5 [hunk 1]
```diff
@@ -21,14 +21,22 @@
...
+        // LayoutLib intentionally omitted to be updated independently.
       ],
       "automerge": false,
-    }
+    },
+    {
+      "description": "LayoutLib shouldn't auto-merge, because updating it is a complex process, see libs/README.md.",
+      "matchPackagePrefixes": [
+        "app.cash.paparazzi:layoutlib-native-",
+        "com.android.tools.layoutlib:layoutlib-api",
```
Comment (TWiStErRob) [inline renovate.json5:37]: @jrodbx please make sure these are correct

File: renovate.json5 [hunk 2]
```diff
@@ -21,14 +21,22 @@
...
-        // LayoutLib intentionally omitted to be updated independently
+        // LayoutLib intentionally omitted to be updated independently.
       ],
       "automerge": false,
-    }
+    },
+    {
+      "description": "LayoutLib shouldn't auto-merge, because updating it is a complex process, see libs/README.md.",
+      "matchPackagePrefixes": [
+        "app.cash.paparazzi:layoutlib-native-",
```
Comment (jrodbx) [inline renovate.json5:36]:
```suggestion
        "app.cash.paparazzi:layoutlib-",
```

**Review by jrodbx — APPROVED** [review]

File: renovate.json5 (hunk 2 above)
Comment (TWiStErRob) [reply renovate.json5:36]: These are the only usages of versions.layoutlib

https://github.com/cashapp/paparazzi/blob/master/gradle/libs.versions.toml#L48-L52

is this for future proofing?

lgtm, commit from UI if you want to go ahead.

Comment (jrodbx) [reply renovate.json5:36]: Yep, future proofing!

**Pull request merged**